config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -315,15 +315,11 @@ periodics:
       - --test-cmd-args=--nodes=600
       - --test-cmd-args=--provider=kubemark
       - --test-cmd-args=--report-dir=/workspace/_artifacts
-      - --test-cmd-args=--testconfig=testing/density/config.yaml
+      # TODO(https://github.com/kubernetes/perf-tests/issues/1007): load test should be used to test high-density.
```

Comment:
Since you're touching it - can you also open an issue to setup regular GCE high-density test (so that we not only have kubemark)?
[I guess we may need bigger nodes or sth, that's why we shouldn't do that in this PR]

Reply:
Done - https://github.com/kubernetes/perf-tests/issues/1028